client/components/Form.jsx

```diff
@@ -0,0 +1,188 @@
+import React from "react";
```

Comment:
Can we call this file something other than "Form". This is the AddVolunteer Form, right? So I'd call it something along the lines of `AddVolunteer` or something more descriptive than Form.

Reply:
renaming it "AddVolunteerForm"